backend/capellacollab/sessions/crud.py

```diff
@@ -56,6 +56,8 @@ def create_session(db: Session, session: DatabaseSession):
     db.add(session)
     db.commit()
     db.refresh(session)
+    session.state = "New"
+    session.last_seen = "UNKNOWN"
```

Comment:
I see. This is indeed a dirty trick to avoid a type case, and appearently it works, since `GetSessionsResponse` acts more like an interface, than an actual (type) class.